### DISCUSSION TITLE
Feature request:  Code snippit box

### DISCUSSION THREAD

**Discussion by dylan-rtfm** [Azure/api-management-developer-portal#2588] [Feature request]
We are publishing sample code within a documentation narrative (cookbooks) but the only decent way to show code within the text is to use a text block and monospace font.  Not great.

On the API pages, in the Try It section, there is highlighted code, including a Copy button.  I would like this on the 'documentation' pages too, as a widget.
![image](https://user-images.githubusercontent.com/94375673/141803980-5db9d645-8178-4b64-a787-aebaed229f8b.png)

Are there any plans to make this (or something similar) as a widget?  The widget set-up could allow the language selection, same languages as in the Try It section.

Thanks.

**Comment by ghost** [comment]
@dylan-rtfm, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
Hi @dylan-rtfm, makes sense, we'll add it.

**Comment by ghost** [comment]
@dylan-rtfm, thank you for requesting this feature.

**Comment by mikebudzynski** [comment]
Loosely related to #1207.
